tests/nested/manual/grade-signed-cloud-init-testkeys/task.yaml

```diff
@@ -145,14 +142,14 @@ execute: |
 
   # gracefully shutdown so that we don't have file corruption
   echo "Gracefully shutting down the nested VM to prepare a simulated attack"
-  boot_id="$(nested_get_boot_id)"
-  nested_shutdown
+  boot_id="$(tests.nested boot-id)"
+  tests.nested vm stop
 
   # replace the seed.iso with the new attacker iso
   mv seed2.iso seed.iso
 
   echo "Restarting nested VM with attacker cloud-init CD-ROM drive"
-  nested_force_start_vm
+  tests.nested vm start
```

Comment:
AFAIU start is really using "force_start" underneath, so this is fine.

Reply:
yes, we should always use tests.nested vm start, it makes sure the start is done and the control to the test is back once the VM is ready.
Thanks for the review